project.mml

```diff
@@ -1411,6 +1411,23 @@ Layer:
     properties:
       cache-features: true
       minzoom: 12
+  - id: stop-positions
```

Comment:
Can you name it `train-stop-positions` or something similar to make it clear it's not all stop positions?